admin-dev/themes/new-theme/.webpack/common.js

```diff
@@ -120,11 +164,11 @@ module.exports = {
     ],
   },
   plugins: [
+    new ExtractTextPlugin('theme.css'),
     new CleanWebpackPlugin(['public'], {
       root: path.resolve(__dirname, '../'),
       exclude: ['theme.rtlfix']
     }),
-    new VueLoaderPlugin(),
```

Comment:
what is the role of the VueLoaderPlugin?

Reply:
This one is not compatible with webpack2, there is already a rule for vue-loader